docs/environments/setup.md

```diff
@@ -106,8 +107,21 @@ EF-CMS currently has both the concept of a deployment at a domain as well as a n
       - e.g. `cd iam/terraform/environment-specific/main && ../bin/deploy-app.sh $ENVIRONMENT`
     - mention your `DYNAMSOFT_PRODUCT_KEYS_$ENVIRONMENT`
 8. Run the `deploy-app.sh` command that you just added to `SETUP.md`.
-10. Modify `.circleci/config.yml` to add `$ENVIRONMENT` to every step under `build-and-deploy` where you want it to be built and deployed.
-11. Update CircleCI to have all the new environment variables needed.
+9. Modify `.circleci/config.yml` to add `$ENVIRONMENT` to every step under `build-and-deploy` where you want it to be built and deployed.
+10. Update CircleCI to have all the new environment variables needed.
+11. Setting up Users
+
+    1. The new environment will require an Admin account for creating users. Run the following command to create the admin account. NOTE: this script also deactivates this user.
+
+        ```bash
+        node shared/admin-tools/user/setup-admin.js
+        ```
+
+    2. If the new environment is a test environment, we have a script to setup test users for the various roles throughout the application. It activates and then deactivates the Admin user.
+
+        ```bash
+        node shared/admin-tools/user/setup-test-users.js
+        ```
```

Comment:
👏 Love the "enable, perform, disable" attitude, and that it’s encapsulated in a single command to do all three.